client/tests/configuration_test.py

```diff
@@ -181,6 +186,21 @@ def test_init(
         self.assertEqual(configuration.search_path, ["simple_string/"])
         self.assertEqual(configuration.file_hash, "HASH")
 
+        expanduser.side_effect = [
+            "/home/user/simple_string_searchpath",
+            "/home/user/simple_string_searchpath"
+        ]
+        json_load.side_effect = [
+            {
+                "search_path": "~/simple_string_searchpath",
```

Comment:
`search_path` is meant to be a list of search path elements - could you rewrite the test in the form of
`"search_path": ["~/simple", {"root": "~/simple", "subdirectory": "subdir"}]`?

Reply:
Yes, thanks - it's a very rough first pass at a test case, so I definitely goofed that. I'm still planning to add more tests for the other fields (`typeshed`, `binary`, etc) as well